charts/gardener-extension-shoot-lakom-service/templates/deployment.yaml

```
@@ -45,18 +45,20 @@ spec:
       {{- if gt (int .Values.replicaCount) 1 }}
       affinity:
         podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchExpressions:
-              - key: app.kubernetes.io/name
-                operator: In
-                values:
-                - gardener-extension-{{ .Values.serviceName }}
-              - key: app.kubernetes.io/instance
-                operator: In
-                values:
-                - {{ .Release.Name }}
-            topologyKey: "kubernetes.io/hostname"
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                  - {{ .Values.name }}
```

```suggestion
                  - gardener-extension-{{ .Values.serviceName }}
```

Isn't this the correct value? I do not see {{ .Values.name }} being set as a label on the pods.

---

Good catch!